osu.Game/Overlays/NotificationOverlay.cs

```diff
@@ -92,8 +92,8 @@ private void load()
                                     RelativeSizeAxes = Axes.X,
                                     Children = new[]
                                     {
-                                        new NotificationSection(AccountsStrings.NotificationsTitle, new[] { typeof(SimpleNotification) }, "Clear All"),
-                                        new NotificationSection(@"Running Tasks", new[] { typeof(ProgressNotification) }, @"Cancel All"),
+                                        new NotificationSection(AccountsStrings.NotificationsTitle, new[] { typeof(SimpleNotification) }, NotificationsStrings.ClearAll),
```

Comment:
Can use existing web translation (seen here):
![image](https://user-images.githubusercontent.com/35318437/212420844-70dae126-7c54-42ec-844a-f67618f85cae.png)

```suggestion
                                        new NotificationSection(AccountsStrings.NotificationsTitle, new[] { typeof(SimpleNotification) }, NotificationsStrings.MarkRead(NotificationsStrings.FiltersDefault)),
```

Class names are going to conflict though.

Reply:
I'll be honest, I would be kind of cautious with reusing translations like that. You can't know whether all usages are going to be in the exact same context with that sort of reuse. The necessity of having to auto-fill `FiltersDefault` here is also making me a little worried.

I guess we can go with it until someone reports a problem, though.

Reply:
Please don't use it from the web, in Russian this has another context.